create-anaconda-payload

```diff
@@ -55,8 +55,48 @@ lvm2
 
 KICKSTART_PATH = "/tmp/payload.ks"
 
+WEBUI_PAYLOAD_SPEC = """
+Name: webui_payload
+Version: 1
+Release: 1
+Summary: Inject what's needed to Web UI image.
+License: FIXME
 
-def build_payload(image: str, output: str) -> None:
+%description
+Inject what's needed to Web UI image.
+
+%source0 webui-payload.tar.gz
+
+%prep
+
+    # we have no source, so nothing here
+    ls -lah
+    cp /root/webui-payload.tar.gz .
+
+%build
+cat > custom_interactive-defaults.ks <<EOF
+liveimg --url=\"file://live.tar.xz\"
+EOF
+
+%install
+mkdir -p %{buildroot}/usr/bin/
+mkdir -p %{buildroot}/usr/share/anaconda/
+
+install -m 755 webui-payload.tar.gz %{buildroot}/live.tar.xz
+install -m 755 custom_interactive-defaults.ks %{buildroot}/usr/share/anaconda/custom_interactive-defaults.ks
+
+%files
+/live.tar.xz
+/usr/share/anaconda/custom_interactive-defaults.ks
+
+%posttrans
+cp /usr/share/anaconda/custom_interactive-defaults.ks /usr/share/anaconda/interactive-defaults.ks
```

Comment:
This needs an explanation; scripts are always brittle. Why not just create a static symlink in `%install` and ship it right in the rpm? Or name the file like that right away?